src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/hmi/on_bc_system_capability_updated_notification_from_hmi.cc

```diff
@@ -36,6 +36,7 @@
 #include "application_manager/message_helper.h"
 #include "interfaces/HMI_API.h"
 #include "interfaces/MOBILE_API.h"
+#include "sdl_rpc_plugin/extensions/system_capability_app_extension.h"
```

Comment:
Please remove sdl_rpc_plugin/, and ajust cmake to use "extensions/system_capability_app_extension.h"

Reply:
@LuxoftAKutsan Done in bbdf770e46270a5cc6f246d411401e8e0e8bfc4e